kraft.yaml

```
@@ -1,7 +1,7 @@
 specification: '0.5'
```

```suggestion
---
specification: '0.5'
```

Compatible with the new syntax